File: Program's_Contributed_By_Contributors/Python_Programs/selection_sort.py
def selection(a,n):
    for i in range(n):
        for j in range(i+1,n):
            if a[i]>a[j]:
                a[i],a[j]=a[j],a[i]
    
    for i in range(n):
        print(a[i],end=" ")
    return 
a=[5,6,7,9,3,4,2]
n=len(a)
selection(a,n)

File: Program's_Contributed_By_Contributors/C programming/selectionSort.c
#include <stdio.h>

void selection(int arr[],int size);
void printArr(int arr[],int size);
void swap(int *x,int *y);

int main(){

    int arr1[] = {1,5,9,4,3};
    int arr2[] = {20,15,10,2,12};
    size_t size1 = sizeof(arr1)/sizeof(arr1[0]);
    size_t size2 = sizeof(arr2)/sizeof(arr2[0]);
    selection(arr2,size2);
    printArr(arr2,size2);
}

void selection(int arr[],int size){
    int min_idx;
    int k;
    for(int i = 0; i<size ;i++){
        min_idx = i;
        for(int j = i+1; j<size ;j++){
            if(arr[j]<arr[min_idx]){
                min_idx = j;
            }
        }
        swap(&arr[i],&arr[min_idx]);
    }
}

void swap(int *x,int *y){
    int temp = *x;
    *x = *y;
    *y = temp;
}

void printArr(int arr[],int size){
    for(int i=0;i<size;i++){
        printf("%d ",arr[i]);
    }
    printf("\n");
}